python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -128,10 +132,16 @@ def measure(self, measRecord, exposure):
         xpy = Ixx + Iyy
         xmy2 = xmy*xmy
         xy2 = Ixy*Ixy
-        a = np.sqrt(0.5 * (xpy + np.sqrt(xmy2 + 4.0*xy2)))
-        L = 2.0*a
+        a2 = 0.5 * (xpy + np.sqrt(xmy2 + 4.0*xy2))
+        sigma = exposure.getPsf().getSigma()
+
+        L, results = self.findLength(a2, sigma*sigma)
```

Comment:
We try to stay away from single variable names. I get this this is probably from a formula (as many of these are), please add to the note section to where people can see the formulas used edc